scripts/gen_vimdoc.py

```diff
@@ -317,7 +321,9 @@ def render_node(n, text, prefix='', indent='', width=62):
         text += ' [verbatim] {}'.format(get_text(n))
     elif n.nodeName == 'listitem':
         for c in n.childNodes:
-            text += indent + prefix + render_node(c, text, indent=indent+(' ' * len(prefix)), width=width)
+            text += indent + prefix + \
+                render_node(c, text, indent=indent +
+                            (' ' * len(prefix)), width=width)
```

Comment:
autopep8 should put operators at start-of-line, not end.

Reply:
Used black (with single quotes) for that now.